packages/desktop-client/src/components/Notifications.tsx

```diff
@@ -164,17 +172,21 @@ function Notification({
               style={{
                 backgroundColor: 'transparent',
                 border: `1px solid ${
-                  positive ? colors.g5 : error ? colors.r4 : colors.y3
+                  positive
+                    ? theme.noticeAccent
+                    : error
+                    ? theme.errorAccent
+                    : theme.warningAccent
                 }`,
                 color: 'currentColor',
                 fontSize: 14,
                 flexShrink: 0,
                 '&:hover, &:active': {
                   backgroundColor: positive
-                    ? colors.g9
+                    ? theme.altNoticeBackground
                     : error
-                    ? colors.r10
-                    : colors.y9,
+                    ? theme.altErrorBackground
+                    : theme.altWarningBackground,
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏same